refinery/analysis_manager/tasks.py

```diff
@@ -3,7 +3,7 @@
 
 @author: nils
 '''
-import urlparse
+import urllib.parse
```

Comment:
just `import urllib` is sufficient

Reply:
i tried this:
```import urllib
urllib.parse('foo')
Traceback (most recent call last):
  File "<input>", line 1, in <module>
TypeError: 'module' object is not callable```

i also don't want to `from urllib import parse` (or its ilk in `urllib` since those words are so common)